Code/Editor/TrackView/SequenceBatchRenderDialog.cpp

```diff
@@ -639,6 +646,11 @@ void CSequenceBatchRenderDialog::OnFPSChange(int itemIndex)
     CheckForEnableUpdateButton();
 }
 
+void CSequenceBatchRenderDialog::OnDirectorChange(int)
```

Comment:
update: Please include parameter name here

Reply:
done

Reply:
Thank you! 👍